docs/modules/ROOT/pages/custom-attributes.adoc

```diff
@@ -0,0 +1,75 @@
+= Custom Attributes
+
+Custom Attributes are a string of space-separated `_attribute_=_value_` pairs added to the header of each AsciiDoc page viewed by the extension before it is rendered.
```

Comment:
It's not completely accurate. We might want to say that it's possible to define custom attributes as a space separated list of `attribute=value` pairs.

Custom attributes are passed to the Asciidoctor processor using the API (highest precedence).

It's also worth noting that we can declare attributes using query parameters:

```
file:///path/to/file.adoc?toc=left
```

I believe that attributes defined as query param have higher precedence but I need to check the code.

Reply:
Also, you can declare built-in attributes to change the default value.

Reply:
Agree.

I'll rework the wording.

I checked the code at converter.js. The extension: 

1. applies its built in attribute settings, 
2. does work with file extensions, 
3. applies attributes from the Extension's Custom Attributes string, 
4. followed by the URL query parameters,
5. and removes attribute kroki-fetch-diagram that doesn't make sense in a browser environment.

Reply:
Fixed.